phive.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<phive xmlns="https://phar.io/phive">
+  <phar name="phpcbf" version="^3.6.0" location="./tools/phpcbf.phar" copy="false" installed="3.6.0"/>
+  <phar name="phpcs" version="^3.6.0" location="./tools/phpcs.phar" copy="false" installed="3.6.0"/>
+</phive>
```

Comment:
I'm not familiar with PHIVE. What's the benefit over installing PHPCS via Composer? Will Dependabot be able to update PHIVE dependencies?

Reply:
[PHIVE](https://phar.io/) is a package manager for PHP development tools that installs the tools as stand-alone, GPG-signed PHARs (i.e., a PHAR contains the corresponding tool with all its dependencies). This helps avoiding dependency hell - both inter-tools as well as between a tool and the project in which it is used. (Dependency hell is e.g., when package A requires package X in version < 3, while package B requires package X in version >= 3.) (This was a real problem in some of my projects.)

In addition, this allows tools like [PHPStan](https://packagist.org/packages/phpstan/phpstan) to be used in their current version (which requires PHP >= 7.1) without breaking Composer-installability of the project in lower PHP versions. (This also is a real problem in some of my projects.)

PHIVE is intended to be used for development tools only, and the PHIVE-installed tools are not expected to be shipped with the library package (hence the entries in the `.gitattributes), and they are also not expected to be deployed to production systems.

Dependabot currently does not update the PHIVE-installed tools yet.

Reply:
Got it. Makes sense to me, although we've managed to work around this issue in [amp-wp](https://github.com/ampproject/amp-wp) which uses PHPCS and PHPStan and supports PHP 5.6+.